doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,243 @@
+Use the :kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_COUNT` Kconfig option to set the value for this state.
+
+SAR Unicast Retransmissions Without Progress Count
+==================================================
+
+This state holds a value that defines the maximum number of retransmissions of a segmented message to a unicast address that will be sent if no acknowledgment
+was received during the timeout, or if an acknowledgment with already confirmed segments was received.
+Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_WITHOUT_PROG_COUNT` to set the maximum number of retransmissions.
+
+SAR Unicast Retransmissions Interval Step
+=========================================
+
+The value of this state controls the interval step used for delaying the retransmissions of unacknowledged segments of a segmented message to a unicast address.
+The interval step is measured in milliseconds.
+
+Use the :kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_STEP` Kconfig option to set a value.
+This value is then used to calculate the interval step using the following formula::
+
+(CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_STEP + 1) * 25 ms
+
+SAR Unicast Retransmissions Interval Increment
+==============================================
+
+SAR Unicast Retransmissions Interval Increment holds a value that controls the interval increment used for delaying the retransmissions of unacknowledged
+segments of a segmented message to a unicast address. The increment is measured in milliseconds.
+
+Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_INC` to set a value.
+The Kconfig option value is used to calculate the increment using the following formula::
+
+(CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_INC + 1) * 25 ms
+
+SAR Multicast Retransmissions Count
+===================================
+
+The state holds a value that controls the total number of retransmissions of a segmented message to a multicast address.
+Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_MULTICAST_RETRANS_COUNT` to set the value for this state.
+
+SAR Multicast Retransmissions Interval Step
+===========================================
+
+This state holds a value that controls the interval between retransmissions of all segments in a segmented message to a multicast address.
+The interval is measured in milliseconds.
+
+Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_MULTICAST_RETRANS_INT` to set a value that is used to calculate the interval
+using the following formula::
+
+(CONFIG_BT_MESH_SAR_TX_MULTICAST_RETRANS_INT + 1) * 25 ms
+
+SAR Discard Timeout
+===================
+
+The value of this state defines the time since the last successfully received segment before giving up the reception of a segmented message.
+Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_RX_DISCARD_TIMEOUT` to set the value for this state.
```

Comment:
What unit is this given in? (seconds/milliseconds)

Reply:
seconds